.github/ISSUE_TEMPLATE/flaky_test_report.md

```diff
@@ -0,0 +1,54 @@
+---
+name: Flake Report
+about: Report a flaky test in KubeVirt
+title: 'test_name'
+labels: kind/flake
+assignees: ''
+---
+
+# What happened
+
+<!-- insert test name -->
+Flaky test detected: {test_name} [1]
+
+<!-- labels for flaky tests -->
+/kind flake
+/priority critical-urgent
+
+<!-- sig assignment
+     all tests contain a sig identifier, please assign the corresponding SIG to the issue, i.e.
+     for a test name containing [sig-compute] or [sig-operator]
+-->
+/sig compute
+
+<!-- note: the flakefinder url needs to be a stable one, i.e. instead of the moving latest report use any with a date instead -->
+[1]: {flakefinder_url}
+
+## Additional context
+Add any other context about the problem here.
+
+# Flake Action Plan
+
+**As the assignee**, **thoroughly review the issue** and put the resulting report as comment into this issue.
+
+## Then decide on one of the following actions:
+
+### **The flake is a critical bug**
+
+* Add label commenting `/triage accepted`
```

Comment:
I guess that a reviewer should assign this label?

Reply:
As stated above, the assignee who executes upon the issue, should add this label. Do you think I should change the wording to make that more explicit?

Reply:
Seems that I've misunderstood this label's purpose.
Reading its [docs](https://github.com/kubevirt/kubevirt/labels?q=accepted), it says `Indicates an issue or PR is ready to be actively worked on`.

Makes sense to me now, thank you!